dags/sentinel-2_data_transfer.py

```diff
@@ -71,100 +94,159 @@ def africa_tile_ids():
 
     return africa_tile_ids
 
-def publish_to_sns_topic(message, attribute):
+
+def publish_to_sns_topic(message):
     """
     Publish a message to a SNS topic
     param message: message body
     """
 
-    sns_hook = AwsSnsHook(aws_conn_id=dag.default_args['africa_conn_id'])
+    body = json.loads(message.body)
+    attributes = body.get("MessageAttributes", {})
+    attributes["product"] = attributes.pop("collection")
+    attributes["product"] = "s2_l2a"
+
+    metadata = json.loads(body.get("Message"))
+    metadata_str = json.dumps(metadata)
```

Comment:
This is loading a message from a string to a dict and then dumping it as a string again. Why?

Reply:
Have cleaned this.